packages/DropdownMenu/src/Confirmation/Confirmation.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import ContentContainerStyled from "../ContentContainer.styles";
+import ConfirmationStyles from "./Confirmation.styles";
+
+const { func, string } = PropTypes;
+
+const propTypes = {
+  buttonSize: string,
+  confirmButtonType: string,
+  confirmLabel: string.isRequired,
+  description: string,
+  onCancel: func.isRequired,
+  onConfirm: func.isRequired,
+  title: string.isRequired,
```

Comment:
😬 Can we also rename this to `header` or `heading`? I think as a rule we should avoid `title`, which is a valid html attribute, for a prop name.

Reply:
gone with heading for now